Mathlib/Algebra/Group/TypeTags.lean

```diff
@@ -76,6 +76,11 @@ protected lemma «forall» {p : Additive α → Prop} : (∀ a, p a) ↔ ∀ a,
 @[simp]
 protected lemma «exists» {p : Additive α → Prop} : (∃ a, p a) ↔ ∃ a, p (ofMul a) := Iff.rfl
 
+/-- Recursion principle for `Additive`, supported by `cases` and `induction`. -/
+@[elab_as_elim, cases_eliminator, induction_eliminator]
```

Comment:
Is there a need to add @[eliminator] here?

Reply:
No, because `unknown attribute [eliminator]`; you must have hallucinated it existing.